bin/ci-import

```diff
@@ -28,9 +28,9 @@ waitingPid=$!
 wait "$waitingPid"
 waitingReturnCode=$?
 # clean up PHP processes
-kill "$workerPid" "$queueWatchPid";
+kill "$workerPid" "$queueWatchPid" || true;
```

Comment:
this is a smell, they shouldn't terminate by themselves as they are long-running processes